plugin/plugin.go

```diff
@@ -37,7 +37,7 @@ func (p *plugin) Find(ctx context.Context, req *secret.Request) (*drone.Secret,
 	// to retrieve the secret at the requested path.
 	params, err := p.find(path)
 	if err != nil {
-		return nil, errors.New("secret not found")
```

Comment:
can you please provide the human readable message and the err being returned.

Reply:
Thanks, I'm not fluent in go and just hacked something together which worked "good enough" to figure out the root issue.
I applied your change! 🙂